config/application.rb

```diff
@@ -126,7 +126,7 @@ def self.config
   ENABLE_DEVELOPMENT_LOG_IN = Rails.env.local?
   MAIL_SENDER = "RubyGems.org <no-reply@mailer.rubygems.org>".freeze
   PAGES = %w[
-    about data download security sponsors
+    about data download security sponsors supporters
```

Comment:
Do we need to remove the `sponsors` page from this list? 

Additionally, to preserve SEO, we should add a redirect in the route for: `/sponsors => /supporters`

Reply:
We should remove sponsors so I simply missed that piece in this file.  Sure, preserving SEO makes sense to me.